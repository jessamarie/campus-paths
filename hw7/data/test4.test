r
Troy Building
INVALID
r
INVALID
Troy Building
r
INVALID1
INVALID2
r
INVALID
INVALID
q