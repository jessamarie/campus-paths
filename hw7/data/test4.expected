First building id/name, followed by Enter: Second building id/name, followed by Enter: 
Unknown building: [INVALID]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [INVALID]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [INVALID1]
Unknown building: [INVALID2]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [INVALID]
